src/EventService.Business/Commands/Event/CreateEventCommand.cs

```diff
@@ -71,11 +85,23 @@ private async Task SendInviteEmailsAsync(List<Guid> userIds, string eventName)
 
   public async Task<OperationResultResponse<bool>> ExecuteAsync(CreateEventRequest request)
```

Comment:
Correct me if i'm wrong, organaizer of event must be added to participants list when u create an event

Reply:
Organizer becomes participant in mapper